### PR TITLE
Use notify.js to create notifications

### PR DESCRIPTION
**Pull request by bierik**
Fixes https://github.com/4teamwork/opengever.core/issues/1340

Depends on https://github.com/4teamwork/plonetheme.teamraum/pull/388

Demo:

notify

### CODE REVIEW

**Comment by phgross** [comment]
:+1:

**Comment by bierik** [comment]
@deiferni updated

**Comment by bierik** [comment]
@deiferni updated... made notificaions to display from top to down

File: opengever/base/upgrades/profiles/4601/jsregistry.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<object name="portal_javascripts" meta_type="JavaScripts Registry">
+
+    <javascript cacheable="True" compression="safe" cookable="True" expression=""
+                enabled="True" id="++resource++opengever.base/notify.js" inline="False"
+                insert-after="++resource++plone.app.jquery.js"
+                />
+
+    <javascript id="++resource++opengever.base/MessageFactory.js"
```
Comment (deiferni) [inline opengever/base/upgrades/profiles/4601/jsregistry.xml]: What about cookable, cacheable, ... are these enabled by default?

**Comment by deiferni** [comment]
One small question, otherwise :+1: 

(One more comment on https://github.com/4teamwork/plonetheme.teamraum/pull/388)

File: opengever/base/upgrades/profiles/4601/jsregistry.xml (hunk 1 above)
Comment (bierik) [inline opengever/base/upgrades/profiles/4601/jsregistry.xml]: MessageFactory is already defined i just extend it with the ```insert-after``` attribute

**Comment by bierik** [comment]
@deiferni updated for sticky notifications

File: opengever/base/upgrades/profiles/4601/jsregistry.xml (hunk 1 above)
Comment (deiferni) [inline opengever/base/upgrades/profiles/4601/jsregistry.xml]: ah :+1:

**Comment by deiferni** [comment]
:+1:

**Pull request merged**